pos_payment_method_cashdro/static/src/js/screens.js

```diff
@@ -0,0 +1,243 @@
+                this.$('.next').addClass('oe_hidden');
+                this.$('.in_cashdro_transaction').removeClass('oe_hidden');
+                this.$('.cashdro-transaction-start').addClass('oe_hidden');
+                this.$('.cashdro-transaction-cancel').removeClass('oe_hidden');
+            } else {
+                this.$('.next').removeClass('oe_hidden');
+                this.$('.in_cashdro_transaction').addClass('oe_hidden');
+                this.$('.cashdro-transaction-start').removeClass('oe_hidden');
+                this.$('.cashdro-transaction-cancel').addClass('oe_hidden');
+            }
+        },
+
+        cashdro_send_payment_request: function () {
+            // The payment is done in three concatenated steps:
+            // 1. The POS send a payment request, to which the Cashdro respondes
+            //    with an operation id.
+            // 2. Then the POS has to acknowledge that such operation id has
+            //    been received.
+            // 3. Once acknowledged the POS has to send a payment request to the
+            //    cashdro. Once the Cashdro responses with a "F" state (for
+            //    finished) we'll get the response and fill the tendered money
+            //    for the payment line.
+            var _this = this;
+            var order = this.pos.get_order()
+            var payment_line = order.selected_paymentline;
+            // Cashdro treats decimals as positions in an integer we also have
+            // to deal with floating point computing to avoid decimals at the
+            // end or the drawer will reject our request.
+            var amount = parseInt(order.get_due(payment_line).toFixed(2) * 100);
+            var url = this._cashdro_payment_url({"amount": amount});
+            var operation_id = "";
+            var request = this._cashdro_request(url)
+                .then(function (res) {
+                    // It comes handy to log the response from the drawer, as
+                    // we can diagnose the right sytmoms for each issue
+                    console.log(res);
+                    operation_id = res.data;
+                    _this.pos.get_order().cashdro_operation = operation_id;
+                    // Acknowledge the operation
+                    var ack_url = _this._cashdro_ack_url(operation_id);
+                    return _this._cashdro_request(ack_url);
+                })
+                .then(function (res_ack) {
+                    // Validate the operation
+                    console.log(res_ack);
+                    var ask_url = _this._cashdro_ask_url(operation_id);
+                    return _this._cashdro_request_payment(ask_url);
+                })
+                .then(function (operation_data) {
+                    // This might be too verbose, but it helps a lot to diagnose
+                    // issues and their reasons.
+                    console.log(operation_data);
```

Comment:
If you want to keep these, at least do it only in debug mode.
It would help to add a descriptive string too

```suggestion
                    // This might be too verbose, but it helps a lot to diagnose
                    // issues and their reasons.
                    if (window.odoo.debug) {
                        console.log("Cashdro operation data", operation_data);
                    }
```